src/cmd.js

```diff
@@ -258,6 +291,71 @@ function action(actionToDispatch) {
   });
 }
 
+function clearTimeoutCmd(timerId) {
+  return run(clearTimeout, { args: [timerId] });
```

Comment:
this will actually cause a weird error `TypeError: Illegal invocation`

I don't totally understand why, but it has to do with the context of the window object when you call the function.

The way to get around it is to make a helper function and call it in there. Here is what I have in a project of mine.

```js
export function clearDelayedEffect(isTimeout, id) {
  const method = isTimeout ? clearTimeout : clearInterval;
  method(id);
}
```

and then you can just return here

```js
return run(clearDelayedEffect, {args: [false, timerId]})
```

you could also do 2 separate helpers to avoid the extra param. doesn't really matter

Reply:
The `.apply()` in https://github.com/redux-loop/redux-loop/pull/242/ fixed the `TypeError: Illegal invocation`.